docs/FAQ.md

```diff
@@ -1,11 +1,159 @@
 # TrenchBoot FAQ
 
-1. [Why does TrenchBoot use an intermediate launcher?](#1-why-does-trenchboot-use-an-intermediate-launcher)
-2. [What are the benefits of measurement over signature validation?](#2-what-are-the-benefits-of-measurement-over-signature-validation)
-3. [What do I need to incorporate TrenchBoot into my system?](#3-what-do-i-need-to-incorporate-trenchboot-into-my-system)
-4. [Where do I start if I want to help with contributions?](#4-where-do-i-start-if-i-want-to-help-with-contributions)
+1. [Why use Trenchboot?](#1-why-use-trenchboot-background-info)
+2. [How does TrenchBoot work?](
+    #2-how-does-trenchboot-work-trenchboot-architecture)
+3. [How to use Trenchboot?](#3-how-to-use-trenchboot-practical-usage)
+4. [Why does TrenchBoot use an intermediate launcher?](
+    #4-why-does-trenchboot-use-an-intermediate-launcher)
+5. [What are the benefits of measurement over signature validation?](
+    #5-what-are-the-benefits-of-measurement-over-signature-validation)
+6. [What do I need to incorporate TrenchBoot into my system?](
+    #6-what-do-i-need-to-incorporate-trenchboot-into-my-system)
+7. [Where do I start if I want to help with contributions?](
+    #7-where-do-i-start-if-i-want-to-help-with-contributions)
+8. [Where to find the sources?](#8-trenchboot-sources)
+9. [Various GitHub and further documentation links](
+    #9-various-github-and-further-documentation-links)
 
-## 1. Why does TrenchBoot use an intermediate launcher?
+## 1. Why use Trenchboot? (background info)
```

Comment:
`TrenchBoot`, here and everywhere in this file.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/41/commits/671b0dd276e69a1846fe77e6de708fb34284a3fb